### DISCUSSION TITLE
Magic connector / social login

### DISCUSSION THREAD

**Discussion by kesar** [wevm/wagmi#401] [Show and tell]
We have created an external connector to use magic as login for social / email

https://github.com/EveripediaNetwork/wagmi-magic-connector

https://www.npmjs.com/package/@everipedia/wagmi-magic-connector

Feel free to give us feedback! ❤️

**Comment by Michael679089** [comment]
This package has been deprecated

Author message:
We moved package to @magiclabs/wagmi-connector ! Please switch to continue receiving updates
